esx_service/utils/error_code.py

```diff
@@ -86,6 +93,8 @@ class ErrorCode:
 
     ErrorCode.DEFAULT_DS_NOT_SET : "Default datastore is not set",
     ErrorCode.DS_NOT_EXIST : "Datastore {0} does not exist",
+    ErrorCode.INIT_NEEDED: "Please init configuration in vmdkops_admin before trying to change it",
```

Comment:
Even better: "Please init configuration using 'vmdkops_admin config init' before trying to change it"